babel.config.js

```diff
@@ -7,29 +7,24 @@ const isCalypsoClient = process.env.CALYPSO_CLIENT === 'true';
 const modules = isCalypsoClient ? false : 'commonjs'; // only calypso should keep es6 modules
 const codeSplit = require( './server/config' ).isEnabled( 'code-splitting' );
 
+const targets = isCalypsoClient
+	? { browsers: [ 'last 2 versions', 'Safari >= 10', 'iOS >= 10', 'ie >= 11' ] }
+	: { node: 'current' };
```

Comment:
This is the version of node where the process is running. Perfect 👍 

As a reader, I had interpreted this as something like "latest stable." That is _not_ the case:

> #### `targets.node`
> 
> `string | "current" | true`.
> 
> If you want to compile against the current node version, you can specify `"node": true` or `"node": "current"`, which would be the same as `"node": process.versions.node`.

https://babeljs.io/docs/en/babel-preset-env#targetsnode